.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1332,16 +1332,14 @@ public class ChatFunctionChoice : IJsonModel<ChatFunctionChoice>, IPersistableMo
         public override readonly string ToString();
     }
     public abstract class ChatMessage : IJsonModel<ChatMessage>, IPersistableModel<ChatMessage> {
```

Comment:
`ChatMessage` should definitely be `abstract`. Should the other five classes (`ChatResponseFormat`, `ChatToolChoice`, `ChatTool`, `ChatToolCall`, and `ChatMessageContentPart`) be `abstract` too?

Reply:
I'd vote "yes."

Reply:
If we don't need to create instances using factory methods and instead we can provide ctors, then non-abstract classes are preferable as users tend to try to instantiate things using ctors first.

Reply:
Sounds good! I will make them abstract as part of a separate PR.